src/lib.rs

```diff
@@ -266,7 +330,17 @@ impl Lzxd {
         // TODO last chunk may misalign this and on the next iteration we wouldn't be able
         // to return a continous slice. if we're called on non-aligned, we could shift things
         // and align it.
```

Comment:
Might need to move this `TODO` inside the call (right where we call `window.past_view`).